### PR TITLE
Add a script for deploying elk via docker compose

### PR DESCRIPTION
**Pull request by ading007**
Testing elk env.
[skip ci]

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+[1 - Docker Compose Overview](https://docs.docker.com/compose/overview/)  
+[2 - Docker compose ELK](https://blogs.vmware.com/cloudnative/2018/07/19/getting-started-with-elastic-stack-on-vsphere-integrated-containers/)
+
+# Environment:
+This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
+
+# Test Steps:
+1. Deploy a new vCenter with 2 ESXi hosts in a cluster
+2. Deploy VIC appliance to the vSphere server 
+3. Issue the following command in the docker voting app folder:
```
Comment (wjun) [inline tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.md]: Please rename "docker voting app folder" to the elk folder

File: tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Resource  ../../resources/Util.robot
+Suite Setup  Deploy Environment
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Variables ***
+${NIMBUS_LOCATION}  sc
+${NIMBUS_LOCATION_FULL}  NIMBUS_LOCATION=${NIMBUS_LOCATION}
+
+*** Keywords ***
+Deploy Environment
```
Comment (wjun) [inline tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot]: I wonder if we could extract this step into a common step so other tests may also use this type of cluster with iscsi?

File: tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.md (hunk 1 above)
Comment (ading007) [reply tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.md]: Currently not needed.

File: tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot (hunk 2 above)
Comment (ading007) [reply tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot]: Currently not needed.

File: tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect -f '{{range $key, $value := .NetworkSettings.Networks}}{{$key}}{{end}}' ${containerName}
+    Log  ${out}
+    Should Not Be Empty  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+*** Test Cases ***
+Compose ELK Test
+    Install VIC Appliance To Test Server  certs=${true}  vol=default  additional-args=--cpu-reservation 1 --cpu-shares normal --memory-reservation 1 --memory-shares normal --endpoint-cpu 1 --endpoint-memory 2048 --base-image-size 0GB --bridge-network-range 172.17.0.0/12 --container-network-firewall vm-network:published --certificate-key-size 2048
```
Comment (luwang-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot]: please change --base-image-size from -GB to 8GB or other size

File: tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    Check Service State  elastic3
+    Check Service State  logstash1
+    Check Service State  logstash2
+    Check Service State  kibana1
+    Check Service State  kibana2
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --skip-hostname-check -f ${CURDIR}/../../../demos/compose/elk-app/docker-compose-elk.yml down
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0    
+
```
Comment (luwang-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot]: Please Add Delete VIC Appliance in the end.

**Pull request merged**